src/makeRobot.js

```diff
@@ -38,7 +38,72 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: `${name}`,
+    wheels: `${wheels}`,
+    version: `${version}`,
```

Comment:
If variables name === key you can use the short method 
```suggestion
    name,
    wheels,
    version,
```